lib/udf_findhotel_utils.rb

```diff
@@ -10,22 +10,25 @@ class UdfFindhotelUtils
             if not room:
               return None
             else:
-              adults = 0
-              children = 0
-              rooms = room.split('|')
-              for r in rooms:
-                ch = r.split(':')
-                adults += int(ch[0])
-                if len(ch) > 1:
-                  children += len(ch[-1].split(','))
+              try:
```

Comment:
@germangh I now handle the exceptions that I expect. 

I only have a question about the difference between writing multiple try/catches vs writing only one try/catch. Which one is the best case?